ext/java/nokogiri/internals/NokogiriHandler.java

```diff
@@ -246,27 +246,25 @@ public void endCDATA() {
         charactersBuilder.setLength(0);
     }
 
-    @Override
-    public void error(SAXParseException ex) {
+    void handleError(SAXParseException spx) {
```

Comment:
I like the diff, but I have a minor ask.  Can you please use `ex` instead of `spx`, this will make the diff easier to read.